experimental/packages/opentelemetry-sdk-metrics-base/src/view/View.ts

```diff
@@ -14,53 +14,107 @@
  * limitations under the License.
  */
 
+import { PatternPredicate } from './Predicate';
+import { AttributesProcessor, FilteringAttributesProcessor } from './AttributesProcessor';
+import { InstrumentSelector } from './InstrumentSelector';
+import { MeterSelector } from './MeterSelector';
 import { Aggregation } from './Aggregation';
-import { AttributesProcessor } from './AttributesProcessor';
+import { InstrumentType } from '../InstrumentDescriptor';
 
-// https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#view
+export type ViewOptions = {
+  // View
+  /**
+   *  If not provided, the Instrument name will be used by default. This will be used as the name of the metrics stream.
```

Comment:
Note, after this change, the options object does contain an `instrumentName` that can be confused with the term "Instrument name" in this statement.

```suggestion
   *  If not provided, the original Instrument name will be used by default. This will be used as the name of the metrics stream.
```

Or can we update the `instrumentName` to be `selectInstrumentName` or something else to clarify that?

Reply:
Thanks for pointing this out. I updated the documentation in [59e69ae](https://github.com/open-telemetry/opentelemetry-js/pull/3066/commits/59e69ae64659b985b246d27a4051b612cc7ae9c5). :slightly_smiling_face: 
I hope that the documentation is sufficient, but I'm not opposed to changing the name too.